modules/c/examples/code_snippets/main.c

```diff
@@ -1712,26 +1711,56 @@ static void docsonly_N1QL_Params(CBLDatabase* argDb)
     // end::query-syntax-n1ql-params[]
 }
 
+static void date_getter(){
+    CBLDatabase* database = kDatabase;
+    CBLCollection* collection = CBLDatabase_DefaultCollection(kDatabase, NULL);
+    CBLError err;
 
+    // tag::date-getter[]
 
-// tag::console-logging-db[]
```

Comment:
Looks like this one has been removed, but that should be OK, as the code snippet is hidden in an `ifndef::is_c`